### PR TITLE
pagination translated

### PR DESCRIPTION
**Pull request by yoheisugiyama**
pagination translated in Japanese

### CODE REVIEW

File: ja/pagination.rst [hunk 1]
```diff
@@ -0,0 +1,395 @@
+.. 
+    Pagination
+
+ページネーション
+##########
```
Comment (markstory) [inline ja/pagination.rst]: Title line length.

File: ja/pagination.rst [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+    building paginated queries easier. In the View
+    :php:class:`~Cake\\View\\Helper\\PaginatorHelper` is used to make the generation
+    of pagination links & buttons simple.
+
+CakePHPにおけるページネーションは、コントローラーにおけるコンポーネントによって提供され、ページ分けされたクエリーをより簡単にビルドできるようにします。ビューの中のView\Helper\PaginatorHelper は、ページネーションのリンクやボタンを作り出すことを容易にすることに使われます。
+
+
+.. 
+    Using Controller::paginate()
+Controller::paginate()の使用
```
Comment (markstory) [inline ja/pagination.rst]: I think you want a newline between the comment and translated text so sphinx doesn't get confuse.

**Comment by yoheisugiyama** [comment]
what do you mean by making the file brand new?

**Comment by yoheisugiyama** [comment]
Can you please merge my commits? I've just amended the documents based on your request. Thank you!!

File: ja/pagination.rst (hunk 2 above)
Comment (yoheisugiyama) [inline ja/pagination.rst]: I've just added a newline as you have mentioned.

**Comment by markstory** [comment]
I noticed a few formatting errors after merging that were fixed in 81c8fc1df5a4160da6a0cd40470b4292ccd5edf0

**Comment by yoheisugiyama** [comment]
Thank you very much!

**Comment by yoheisugiyama** [comment]
Thank you very much! But do you have any idea why the translated document is not reflected in the original site? Would appreciate it if you can check..

http://book.cakephp.org/3.0/ja/controllers/components/pagination.html

Thank you!!

**Pull request merged**